dlpy/utils.py

```diff
@@ -1243,6 +1257,17 @@ def get_txt_annotation(local_path, coord_type, image_size = (416, 416), label_fi
         Specifies the list of filename with XML extension under local_path to be parsed.
         If label_files is not specified, all of XML files under local_path will be parsed .
         Default: None
+    name_file : str, optional
+        Specifies the path of name_file which lists all of the names for the classes in your dataset.
+        If you specify the option, the function will generate txt using index to represent category
+        and space as separator. For example:
+            17 0.539766 0.492976 0.317406 0.345796
+            17 0.557742 0.265619 0.083922 0.087191
+        Otherwise, it will generate the format can be consumed by SAS platform.
```

Comment:
can we have the numbers the same, and indicate that 17 is the internal number assigned for the person category